File: QTD.SPA/src/app/_DtoModels/SchedulesClassses/ClassScheduleTQEMPSettingsVM.ts

export class ClassScheduleTQEMPSettingsVM  {
    id : string ;
    classScheduleId: string;
    tqRequired: boolean;
    releaseOnClassStart: boolean;
    releaseOnClassEnd: boolean;
    specificTime?: number;
    priorToSpecificTime: boolean;
    showTaskSuggestions:boolean;
    showTaskQuestions:boolean;
}

File: QTD2.Infrastructure/Model/ClassSchedule_TQEMPSettings/ClassSchedule_TQEMPSettingsVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.ClassSchedule_TQEMPSettings
{
    public class ClassSchedule_TQEMPSettingsVM
    {
        public int Id { get; set; }
        public int ClassScheduleId { get; set; }
        public bool TQRequired { get; set; }

        public bool ReleaseOnClassStart { get; set; }

        public bool ReleaseOnClassEnd { get; set; }

        public int? SpecificTime { get; set; }

        public bool PriorToSpecificTime { get; set; }
        public bool Active { get; set; }
        public bool ShowTaskSuggestions { get; set; }
        public bool ShowTaskQuestions { get; set; }

        public ClassSchedule_TQEMPSettingsVM() { }
    }
}
